core_backend/requirements.txt

```diff
@@ -20,3 +20,4 @@ redis==5.0.8
 python-dateutil==2.8.2
 gTTS==2.5.1
 google-cloud-storage==2.18.2
+google-cloud-aiplatform==1.47.0
```

Comment:
LiteLLM was complaining that this package wasn't installed! When trying to call vertex AI.

Reply:
Version is from the litellm requirements: https://github.com/BerriAI/litellm/blob/94c069e8698a4a76f47d79726dc26519892bd129/requirements.txt#L16